lms/djangoapps/instructor/views/api.py

```diff
@@ -2846,7 +2846,13 @@ def parse_request_data_and_get_user(request):
     try:
         db_user = get_user_by_username_or_email(user)
     except ObjectDoesNotExist:
-        raise ValueError(_('Student (username/email={user}) does not exist').format(user=user))
+        raise ValueError(_("We can't find the user you've entered. Make sure the username or email address is correct,"
```

Comment:
Also I know it is not part of this particular changeset, but I find the error messages above ('Invalid Json data') and ('Student username/email is required) to be somewhat confusing.  Is it possible to include more context describing the scenarios that would trigger these workflows?